### PR TITLE
Fix/details page tests

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Updated cypress testing for the pokemon details page
- Updated manifest theme and bgColor
- Removed comments, fixed indentations to be consistent

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [ ] local host 3000
- [ ] console.log()
- [ ] dev tools
- [x] cypress testing

# Checklist:

- [ ] My code follows the style guidelines of this project/linted
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by hayleyw7 — APPROVED** [review]
Thanks for updating the tests to reflect the changes in our Components.

File: cypress/integration/pokemon_details_spec.js [hunk 1]
```diff
@@ -1,130 +1,114 @@
...
+      .get('img[alt="mankey image"]')
+      .should('be.visible')
+  });
 
-    it('Should have a search field', () => {
-        cy.get('form')
-          .should('be.visible')     
-    });
+  it('Should show an error message if the user adds the wrong id number', () => {
+    cy.get('input[type="search"]')
```
Comment (hayleyw7) [inline cypress/integration/pokemon_details_spec.js:51]: I appreciate how robust this is.

File: public/manifest.json [hunk 2]
```diff
@@ -11,6 +11,6 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
+  "theme_color": "#CC0000",
```
Comment (hayleyw7) [inline public/manifest.json:14]: Matches perfectly!

**Review by hayleyw7 — APPROVED** [review]

File: public/manifest.json (hunk 2 above)
Comment (joanafbrito) [reply public/manifest.json:14]: Thanks !

**Review by joanafbrito — APPROVED** [review]
Thanks !! Great Job!

**Pull request merged**